noxfiles/docker_nox.py

```diff
@@ -111,35 +114,24 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
             external=True,
         )
 
-    # Allow building the fides-privacy-center:local image directly when
-    # requested (for development purposes)
-    if image == "privacy_center":
-        session.run(
-            "docker",
-            "build",
-            "clients/privacy-center",
-            "--tag",
-            "ethyca/fides-privacy-center:local",
-            external=True,
-        )
-    else:
-        # Build the main ethyca/fides image
-        target = build_matrix[image]["target"]
-        tag = build_matrix[image]["tag"]
-        build_command = (
-            "docker",
-            "build",
-            f"--target={target}",
-            "--platform",
-            build_platform,
-            "--tag",
-            tag(),
-            ".",
-        )
-        if "nocache" in session.posargs:
-            build_command = (*build_command, "--no-cache")
-
-        session.run(*build_command, external=True)
+    # Build the main ethyca/fides image
+    # E.g. of PC command: docker build --target=frontend --platform linux/arm64 --tag ethyca/fides:local-pc .
```

Comment:
This feels redundant since `nox` will log out the command it is running anyway

Reply:
this was leftover from my own notes while testing, will remove!